src/App.scss

```diff
@@ -2,19 +2,19 @@ iframe {
   display: none;
 }
 
-.Person {
-  width: fit-content;
-  margin-bottom: 16px;
-  padding: 8px;
-  border: 1px solid #000;
-  border-radius: 8px;
+// .Person {
+//   width: fit-content;
+//   margin-bottom: 16px;
+//   padding: 8px;
+//   border: 1px solid #000;
+//   border-radius: 8px;
 
-  &__name {
-    margin-top: 8px;
-  }
+//   &__name {
+//     margin-top: 8px;
+//   }
 
-  &__age {
-    color: #00f;
-    font-weight: bold;
-  }
-}
+//   &__age {
+//     color: #00f;
+//     font-weight: bold;
+//   }
+// }
```

Comment:
The styles for the `.Person` class are currently commented out. If these styles are intended for the `Person` component, you should uncomment them to apply the styles. If they are not needed, consider removing them to keep the code clean.